src/components/Missions.js

```diff
@@ -1,8 +1,65 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import Table from 'react-bootstrap/Table';
+import Button from 'react-bootstrap/Button';
+import { useDispatch, useSelector } from 'react-redux';
+import Badge from 'react-bootstrap/Badge';
+import { fetchMissions, joinMission } from '../redux/missions/missionsSlice.js';
```

Comment:
Please, I suggest that you ensure that the name of the file tarry while fetching the function. Please kindly look into it and update it.